doc/pages/administrator/064-setup_astartedefaultingress.md

```diff
@@ -0,0 +1,89 @@
+# Setting up Astarte Default Ingress
+
+Once your Cluster [is up and running](060-setup_cluster.html), to expose it to the outer world you
+need to set up an Ingress. Currently, the only managed and supported Ingress is based upon
+[NGINX](https://nginx.org/en/), and this guide will cover only this specific case.
+
+Please, note that the previously supported Voyager based ingress is deprecated and should not be
+used (see the [Voyager announcement](https://blog.byte.builders/post/voyager-v2021.09.15/) and the
+related Astarte [issue](https://github.com/astarte-platform/astarte/issues/613) for further
+details).
+
+If you want to migrate away from AstarteVoyagerIngress to the new AstarteDefaultIngress, please
+refer to the procedure outlined [here](030-upgrade_100_101.html).
+
+## Prerequisites
+
+Before proceeding with the deployment of the Astarte Default Ingress, the following requirements
+must be fulfilled:
+
+- TLS certificates must be deployed as a secret within the namespace in which Astarte resides (see
+  the [Handling Astarte Certificates](062-handling_certificates.html) section);
+- Astarte must be configured such that TLS termination is handled at VerneMQ level: this can be done
+  simply editing the Astarte resource and, in the `vernemq` section, adding the following:
+  - `sslListener: true`
+  - `sslListenerCertSecretName: <your-tls-secret-name>`
+- [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/) ingress controller must be deployed
+within your cluster. This task can be simply performed by running:
+```bash
+$ helm install ingress-nginx ingress-nginx/ingress-nginx -n ingress-nginx \
+    --set controller.service.externalTrafficPolicy=Local \
+    --create-namespace
+```
+
+## Creating an `AstarteDefaultIngress`
+
+Most information needed for exposing your Ingress have already been given in your main Astarte
+resource. If your Kubernetes installation supports LoadBalancer ingresses (most managed ones do),
+you should be able to get away with the most standard CR:
+
+```yaml
+apiVersion: ingress.astarte-platform.org/v1alpha1
+kind: AstarteDefaultIngress
+metadata:
+  name: adi-example-minimal
+  namespace: astarte
+spec:
+  ### Astarte Default Ingress CRD
+  astarte: astarte
+  tlsSecret: <your-astarte-tls-cert>
+  api:
+    exposeHousekeeping: true
+  dashboard:
+    deploy: true
+    ssl: true
+    host: <your-astarte-dashboard-host>
+  broker:
+    deploy: true
```

Comment:
If one is to use a reserved/fixed loadBalancerIP, there might be some more work to do depending on the cloud provider.
GKE requires one to reserve(or promote) the external IP address (https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address#promote_ephemeral_ip); Azure does not.
Should we add a line on this?

Reply:
I wouldn't expose the detail in the doc at the moment. Anyway, details on this are present in the ADI crd [here](https://github.com/astarte-platform/astarte-kubernetes-operator/blob/release-1.0/apis/ingress/v1alpha1/astartedefaultingress_types.go#L87-L90). @bettio @rbino opinions on this?

Reply:
Nevermind, added info about this.